public/js/components/EditorSearchBar.js

```diff
@@ -91,23 +99,41 @@ const EditorSearchBar = React.createClass({
     this.search(query);
   },
 
-  onKeyUp(e) {
+  traverseResultsPrev(shortcut, e) {
```

Comment:
I like that it's a little clearer as a verb, but I agree it's a little long. I'm fine with how it is though.